<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLColorBarDisplayNodeWidget</class>
 <widget class="qMRMLWidget" name="qMRMLColorBarDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Bar</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/Colors.png</normaloff>../Icons/Colors.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="ctkDynamicSpacer" name="DynamicSpacer">
     <property name="activeSizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="VisibilityLabel">
       <property name="text">
        <string>Visible:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="NumberOfLabelsLabel">
       <property name="text">
        <string>Number of labels:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="DimensionsLabel">
       <property name="text">
        <string>Size (Width and Height):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="MaxNumberOfColorsLabel">
       <property name="text">
        <string>Maximum number of colors:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="DisplayedOnViewLabel">
       <property name="text">
        <string>Displayed on Views:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ctkCheckBox" name="ColorBarVisibilityCheckBox">
       <property name="indicatorIcon">
        <iconset resource="../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
         <normaloff>:/Icons/VisibleOn.png</normaloff>
         <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
       </property>
       <property name="indicatorIconSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="PositionWithinViewLabel">
       <property name="text">
        <string>Position Within View:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="NumberOfLabelsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ctkCoordinatesWidget" name="PositionCoordinatesWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dimension">
        <number>2</number>
       </property>
       <property name="normalized">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="coordinates" stdset="0">
        <string>0.8,0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="VerticalOrientationRadioButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Vectical</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ColorBarOrientationButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="HorizontalOrientationRadioButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Horizontal</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ColorBarOrientationButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="UseColorNamesFromLabelsLabel">
       <property name="toolTip">
        <string>Toggle between using the names of color table colors and the scalar values for the labels in the scalar bar widget.</string>
       </property>
       <property name="text">
        <string>Use color names for labels: </string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="CenterLabelsLabel">
       <property name="toolTip">
        <string>Toggle between centerling names of color table colors on color swatches in the scalar bar widget.</string>
       </property>
       <property name="text">
        <string>Center labels: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ctkCoordinatesWidget" name="WidthHeightCoordinatesWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dimension">
        <number>2</number>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="coordinates" stdset="0">
        <string>0.1,0.8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ColorBarOrientationLabel">
       <property name="text">
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="LabelsGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Labels properties:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="LabelsTextPropertiesGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="LabelsTextLabel">
            <property name="text">
             <string>Format:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="LabelsFontLabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="LabelsSizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LabelsColorLabel">
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="LabelsStyleLabel">
            <property name="text">
             <string>Style:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="LabelsOpacityLabel">
            <property name="text">
             <string>Opacity:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LabelsTextLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="ctkColorPickerButton" name="LabelsColorPickerButton"/>
          </item>
          <item row="2" column="1">
           <widget class="ctkDoubleSlider" name="LabelsOpacitySlider">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="LabelsFontComboBox">
            <item>
             <property name="text">
              <string>Arial</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Courier</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Times</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="LabelsBoldCheckBox">
              <property name="text">
               <string>bold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="LabelsItalicCheckBox">
              <property name="text">
               <string>italic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="LabelsShadowCheckBox">
              <property name="text">
               <string>shadow</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <widget class="ctkDoubleSlider" name="LabelsSizeSlider">
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="MaxNumberOfColorsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox"/>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="CenterLabelCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Toggle between centerling names of color table colors on color swatches in the scalar bar widget.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="UseColorNameAsLabelCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Toggle between using the names of color table colors and the scalar values for the labels in the scalar bar widget.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="TitleGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Title properties:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="TitleTextPropertiesGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="TitleTextLabel">
            <property name="text">
             <string>Text:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="TitleFontLabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="TitleSizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TitleColorLabel">
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="TitleStyleLabel">
            <property name="text">
             <string>Style:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TitleOpacityLabel">
            <property name="text">
             <string>Opacity:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="TitleTextLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="ctkColorPickerButton" name="TitleColorPickerButton"/>
          </item>
          <item row="2" column="1">
           <widget class="ctkDoubleSlider" name="TitleOpacitySlider">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="TitleFontComboBox">
            <item>
             <property name="text">
              <string>Arial</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Courier</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Times</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="TitleBoldCheckBox">
              <property name="text">
               <string>bold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="TitleItalicCheckBox">
              <property name="text">
               <string>italic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="TitleShadowCheckBox">
              <property name="text">
               <string>shadow</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <widget class="ctkDoubleSlider" name="TitleSizeSlider">
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="LogarithmicScaleLabel">
       <property name="text">
        <string>Logarithmic scale:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="LogarithmicScaleCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDynamicSpacer</class>
   <extends>QWidget</extends>
   <header>ctkDynamicSpacer.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLColorBarDisplayNodeWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>DisplayNodeViewComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ColorBarOrientationButtonGroup"/>
 </buttongroups>
</ui>
